zebrad/tests/acceptance.rs

```diff
@@ -2606,3 +2606,83 @@ async fn state_format_test(
     }
     Ok(())
 }
+
+/// Test that fixed responses at zcashd for the `z_getsubtreesbyindex` method are the same as zebrad responses.
+///
+/// Require scyncronized zebrad cache state to query high heights.
+#[tokio::test]
+#[ignore]
+async fn rpc_z_getsubtreesbyindex_zcashd_test_vectors() -> Result<()> {
```

Comment:
This is a note about running this test from the `entrypoint.sh` file, but I can't make suggestions there using GitHub:

We need to run this test with a cached Zebra state, but we're also trying to limit the number of CI jobs we have.

So I suggest adding it as another test in the same job as the existing Zebra RPC test:
https://github.com/ZcashFoundation/zebra/blob/5b0bc700981c39989f46a099d328e00b2166ebac/docker/entrypoint.sh#L88

You could copy that one line and change the test name, then make sure two test are being run in that CI job.